projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -938,12 +941,67 @@ describe('IgxDateRangePicker', () => {
                     fixture.componentInstance.displayFormat = displayFormat;
                     fixture.detectChanges();
 
-                    expect(startInputEditor.inputFormat).toEqual(inputFormat);
+                    expect(startInputEditor.inputFormat).toEqual('MM/dd/yyyy');
                     expect(startInputEditor.displayFormat).toEqual(displayFormat);
-                    expect(endInputEditor.inputFormat).toEqual(inputFormat);
+                    expect(endInputEditor.inputFormat).toEqual('MM/dd/yyyy');
                     expect(endInputEditor.displayFormat).toEqual(displayFormat);
                 });
 
+                it('should set default inputFormat to the start/end editors with parts for day, month and year based on locale ', fakeAsync(() => {
+                    registerLocaleData(localeBg);
+                    registerLocaleData(localeJa);
+
+                    expect(fixture.componentInstance.inputFormat).toEqual(undefined);
+                    expect(dateRange.locale).toEqual('en-US');
+
+                    const startInputEditor = startInput.injector.get(IgxDateTimeEditorDirective);
+                    const endInputEditor = endInput.injector.get(IgxDateTimeEditorDirective);
+                    expect(startInputEditor.inputFormat).toEqual('MM/dd/yyyy');
+                    expect(endInputEditor.inputFormat).toEqual('MM/dd/yyyy');
+
+                    dateRange.locale = 'ja-JP';
+                    fixture.detectChanges();
+                    tick();
+
+                    expect(startInputEditor.inputFormat).toEqual('yyyy/MM/dd');
+                    expect(startInputEditor.nativeElement.placeholder).toEqual('yyyy/MM/dd');
+                    expect(endInputEditor.inputFormat).toEqual('yyyy/MM/dd');
+
+                    dateRange.locale = 'bg-BG';
+                    fixture.detectChanges();
+                    tick();
+
+                    expect(startInputEditor.inputFormat.normalize('NFKC')).toEqual('dd.MM.yyyy г.');
```

Comment:
So this is basically the default functionality of the date time editor directive? IMO, unless the default doesn't match the default type the directive uses (date instead of date-time) I don't even see a reason for the picker to re-implement this at all and the only expectation here would be locale is correctly passed to the inputs and the rest is covered in the directive already?